applications/volumemanager/api/openapi.yaml

```diff
@@ -0,0 +1,118 @@
+openapi: 3.0.0
```

Comment:
Looks pretty generic: I think that moving it to cloud harness as initially designed is still the best option.

Reply:
done